qa/tasks/s3tests.py

```diff
@@ -378,6 +378,10 @@ def run_tests(ctx, config):
         else:
             args += ['REQUESTS_CA_BUNDLE=/etc/pki/tls/certs/ca-bundle.crt']
         attrs = ["not fails_on_rgw", "not lifecycle_expiration"]
+        if not client_config.get('lifecycle_tests'):
+            attrs += ['not lifecycle_expiration', 'not lifecycle_transition']
+        if not client_config.get('cloudtier_tests'):
+            attrs += ['not cloud_transition']
```

Comment:
this caused a failure in the rgw/cloud-transition suite:
> collecting ... collected 780 items / 780 deselected / 0 selected
> ...
> ERROR: InvocationError for command /home/ubuntu/cephtest/s3-tests-client.0/.tox/py/bin/pytest -v -m 'not fails_on_rgw and not lifecycle_expiration and not lifecycle_expiration and not lifecycle_transition and not test_of_sts and not webidentity_test and not fails_with_subdomain and not sse_s3 and cloud_transition' (exited with code 5)

i think it fails because the `cloud_transition` tests are also marked `lifecycle_transition`, so it filters out all test cases